archey/output.py

```diff
@@ -65,14 +68,14 @@ def add_entry(self, module):
         """Append an entry to the list of entries to output"""
         self._entries.append(module)
 
-    def append(self, key, value):
+    def append(self, data):
         """Append a pre-formatted entry to the final output content"""
         self._results.append(
             '{color}{key}:{clear} {value}'.format(
                 color=self._colors_palette[0],
-                key=key,
+                key=data[0],
                 clear=Colors.CLEAR,
-                value=value
+                value=(data[1] or self._configuration('default_strings')['not_detected'])
```

Comment:
Hope this **will** help us in the future to DRY-the-whole-thing :slightly_smiling_face: